impexp-client/src/main/java/org/citydb/gui/modules/database/view/DatabasePanel.java

```diff
@@ -591,6 +649,13 @@ private void setDbConnection(DatabaseConnection databaseConnection) {
 		databaseConnection.setUser(userText.getText());
 		databaseConnection.setPassword(String.valueOf(passwordText.getPassword()));
 		databaseConnection.setSavePassword(passwordCheck.isSelected());
+
+		String workspace = getValue(workspaceCombo);
+		if (databaseTypeCombo.getSelectedItem() == DatabaseType.ORACLE && workspace != null) {
+			databaseConnection.setWorkspace(new Workspace(workspace, timestamp.getDate()));
+		} else {
+			databaseConnection.setWorkspace(null);
```

Comment:
This code looks not correct (see the settings in the screenshot). In this case, the workspace assigned to `databaseConnection` should not be NULL, but a `new Workspace("LIVE", timestamp.getDate())`.
![image](https://user-images.githubusercontent.com/5942971/104243589-140eb080-5461-11eb-909b-f603820d51fd.png)

Reply:
Well, the assumption is that a name is always required if you want to use a timestamp. Does it make sense to set a timestamp on the `LIVE` workspace? If so, then I agree and we must change this behaviour.

Reply:
This should fix it: bc0fb3b264a19d9e41fb93819fd2f131f74f880b. Can you please test again?

Reply:
very good